demisto_sdk/commands/common/logger.py

```diff
@@ -32,9 +32,9 @@ def logging_setup(
     logging.basicConfig()
     l: logging.Logger = logging.getLogger("demisto-sdk")
 
-    levels = {0: logging.INFO, 1: logging.WARNING, 2: logging.DEBUG}
+    levels = {1: logging.INFO, 2: logging.WARNING, 3: logging.DEBUG}
```

Comment:
perhaps worth replacing with a class inheriting both `(int, Enum.enum)`? then it's more readable

Reply:
I don't think we need this at all.
I think we should drop support for (-v, -vv and -vvv) and use just -v to print debug logs, but that's for another PR.

Fixed that because that was a clear bug and the logs didn't work as expected.

Reply:
fyi @tkatzir

Reply:
How big would it be to make the scale linear?
i.e. levels = {3: logging.DEBUG, 1: logging.INFO, 2: logging.WARNING}